code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -436,8 +436,12 @@
 						alert("Unable to use this emote, must be either audible or visible.")
 						return
 					phrase_log.log_phrase("emote", input)
-					message = "<B>[src]</B> [input]"
-					maptext_out = "<I>[regex({"(&#34;.*?&#34;)"}, "g").Replace(input, "</i>$1<i>")]</I>"
+					var/space = should_have_space_before_emote(html_decode(input)) ? " " : ""
+					message = "<B>[src]</B>[space][input]"
+					var/should_have_name_before_emote
+					if (!space)
+						should_have_name_before_emote = "[src]"
+					maptext_out = "<I>[should_have_name_before_emote][regex({"(&#34;.*?&#34;)"}, "g").Replace(input, "</i>$1<i>")]</I>"
```

Comment:
Why is one of these a ternary and one an `if`? either is fine but consistency good